BreakingChanges.md

```diff
@@ -11,6 +13,30 @@ If it is not possible for you to use a C# 7-compatible compiler, we have added a
 
 ---------------
 
+The argument matcher change to support `out` and `ref` mentioned above can also cause a compilation error if argument matchers are used in expression trees (see issue #471):
+
+> CS8153: An expression tree lambda may not contain a call to a method, property, or indexer that returns by reference
+
+Reason: `out` and `ref` support for argument matchers is important to address as it often caused confusion for people now that C# has more support for `out` and `ref`. We are always reluctant to introducing breaking changes, but in this case we estimated the benefit of the feature would outweigh the downside of no longer being able to use matchers in expression trees.
```

Comment:
Is here something wrong with the first sentence near `caused confusion for people now that C# has more support for` or it's just my poor English? 😊 Probably, we could rephrase it a bit to make it simpler...

Reply:
It is my poor English, not yours! 🤣 I'll try to fix it.